test/test_fields.py

```diff
@@ -36,6 +36,14 @@ def test_make_multipart(self):
             'Content-Location: /test\r\n'
             '\r\n')
 
+        field = RequestField('somename', 'data', '')
+        field.make_multipart(content_type='application/octet-stream')
+        self.assertEqual(
+            field.render_headers(),
+            'Content-Disposition: form-data; name="somename"; filename=""\r\n'
+            'Content-Type: application/octet-stream\r\n'
+            '\r\n')
+
```

Comment:
Thanks for writing this test! Can you split it into its own test function please?